authentication/authentication/src/main/resources/application.yaml

```
@@ -34,7 +34,7 @@ spring:
 cloud:
   aws:
     credentials:
-      access-key:
+      access-key: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)